CMakeLists.txt

```diff
@@ -687,6 +694,11 @@ include(cmake/OpenCVFindLibsVideo.cmake)
 include(cmake/OpenCVFindLibsPerf.cmake)
 include(cmake/OpenCVFindLAPACK.cmake)
 include(cmake/OpenCVFindProtobuf.cmake)
+if(WITH_TENGINE)
+  if (X86_64 OR X86 OR ARM OR AARCH64 OR CMAKE_CROSSCOMPILING)
```

Comment:
Move this condition to corresponding `OCV_OPTION`.

Reply:
done